pkg/targets/adapter/ibmmqtarget/adapter.go

```diff
@@ -42,6 +45,9 @@ type ibmmqtargetAdapter struct {
 	logger   *zap.SugaredLogger
 	mqEnvs   *TargetEnvAccessor
 	queue    *mq.Object
+
+	mt *pkgadapter.MetricTag
```

Comment:
It is not necessary to embed this struct because we don't reuse it inside `Start()`.